web/themes/custom/recipe_theme/src/sass/content/_node.landing-page.scss

```diff
@@ -1,5 +1,29 @@
-.landing-page__field-image-hero {
-  margin-top: 3rem;
-  margin-bottom: 3rem;
-  opacity: 0.5;
+.landing-page__body {
+  text-align: center;
+  margin-top: 2.5rem;
```

Comment:
These rules can be simplified as

`margin: 2.5rem 2rem;`

Reply:
Fixed